test/trento_web/views/v1/software_updates_view_test.exs

```diff
@@ -0,0 +1,28 @@
+defmodule TrentoWeb.V1.UserSettingsViewTest do
+  use TrentoWeb.ConnCase, async: true
+
+  import Phoenix.View
+  import Trento.Factory
+
+  alias Trento.SoftwareUpdates.Settings
+  alias TrentoWeb.V1.SoftwareUpdatesView
+
+  describe "renders user_settings.json" do
+    test "should render all the fields" do
+      %Settings{url: url, username: username, ca_uploaded_at: ca_uploaded_at} =
```

Comment:
```suggestion
      %Settings{url: url, username: username, ca_uploaded_at: ca_uploaded_at} = settings =
```
with this we can pass later in the `render(SoftwareUpdatesView, "user_settings.json", settings)` and assert that the output only contains what we expect